docs/Development-Guides/Providers.md

```diff
@@ -13,10 +13,10 @@ title: HRIS
 > **Read:** The `Company`, `Directory`, `Individual`, `Employment`, `Payment` and `Pay Statement` endpoint are read-only endpoints. Providers can be either an [Automated API Provider](../Product-Guides/Automated-Connect-Flow.md) or an [Assisted API Provider](../Product-Guides/Assisted-Connect-Flow.md).
```

Comment:
Please also change `**Last Updated:** 2023-01-20` in line 3 to today's date

Reply:
🤦  Totally forgot that, just added in the update.